CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 ### Added
 
+- `SilentBanner`: added `inline` boolean prop (default false), which will render the banner inline instead of taking full width. ([@driesd](https://github.com/driesd) in [#1220])
+
 ### Changed
 
+- `SilentBanner`: changed to take the full width by default. ([@driesd](https://github.com/driesd) in [#1220])
```

Comment:
Can you add a `[breaking change]` or a  💥 ?

Reply:
Good catch! Mentioned the breaking change in our changelog now.